core/style.js

```diff
@@ -2117,3 +2117,81 @@ CKEDITOR.tools.extend( CKEDITOR.editor.prototype, {
  * @cfg {String/Array/Boolean} [stylesSet='default']
  * @member CKEDITOR.config
  */
+
+/**
+ * Abstract class describing definition of a style.
+ * It is used as one of {@link CKEDITOR.style#constructor}'s arguments.
+ *
+ * Style definition object represents a style as a set of properties describing CSS style rules and HTML attributes.
+ * The {@link CKEDITOR.style} based on such definition can be applied to and removed from selection
+ * through various {@link CKEDITOR.style} methods.
+ *
+ *  	{
+ *			name: 'Special Title',
+ *			element: 'h1',
+ *			attributes: { class: 'my_class' },
+ *			styles: { color: 'red', 'font-size': '16px', 'font-width': 'bold' }
+ *		}
+ *
+ *	More on how editor content styles are handled can be found in the [Styles guide](#!/guide/dev_howtos_styles).
+ *
+ * @class CKEDITOR.style.definition
+ * @abstract
+ */
+
+/**
+ * Defines style type.
+ *
+ * There are three standard style types - {@link CKEDITOR#STYLE_INLINE}, {@link CKEDITOR#STYLE_BLOCK}
+ * or {@link CKEDITOR#STYLE_OBJECT}, each one related to the element used in the style rule andq types of
+ * elements to which specific style can be applied.
+ *
+ * Plugins may define {@link CKEDITOR.style.customHandlers special style handlers} which can be applied in certain situations.
```

Comment:
> which can be applied in certain situations

How about:

> to customize style operations.

?

My point is that it affects all the operations doable with styles:

* checking
* application
* removal